### PR TITLE
UPSTREAM: <drop>: zero-change to kick the build

### PR DESCRIPTION
**Pull request by mfojtik**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: REBASE.openshift.md [hunk 1]
```diff
@@ -228,6 +228,8 @@ This value, among other things, is used by ART to inject appropriate version of
 kubernetes during build process, so it always has to reflect correct level of
 kubernetes.
 
+TODO: Due to a bug in release controller, we need to push the kubernetes tag.
```
Comment (deads2k) [inline REBASE.openshift.md:231]: this is false

Comment (deads2k) [reply REBASE.openshift.md:231]: This is a bug in the kube fork.

**Comment by deads2k** [comment]
merged an actual zero diff change in https://github.com/openshift/kubernetes/pull/1191

**Comment by mfojtik** [comment]
/closeOn Tue, Feb 22, 2022 at 20:20, openshift-ci[bot] ***@***.***> wrote:  
@mfojtik: The following test failed, say /retest to rerun all failed tests or /retest-required to rerun all mandatory failed tests:



Test name
Commit
Details
Required
Rerun command




ci/prow/e2e-aws-serial
b0c6869
link
true
/test e2e-aws-serial



Full PR test history. Your PR dashboard.

Instructions for interacting with me using PR comments are available here.  If you have questions or suggestions related to my behavior, please file an issue against the kubernetes/test-infra repository. I understand the commands that are listed here.

**Pull request closed without merging**